src/cohort/cohort.controller.ts

```
@@ -202,49 +202,49 @@ export class CohortController {
   @UseFilters(new AllExceptionsFilter(APIID.COHORT_DELETE))
-  @Delete("/delete/:cohortId")
-  @ApiOkResponse({ description: "Cohort has been deleted successfully." })
-  @ApiBadRequestResponse({ description: "Bad request." })
-  @ApiInternalServerErrorResponse({ description: "Internal Server Error." })
+  @Delete('/delete/:cohortId')
+  @ApiOkResponse({ description: 'Cohort has been deleted successfully.' })
+  @ApiBadRequestResponse({ description: 'Bad request.' })
+  @ApiInternalServerErrorResponse({ description: 'Internal Server Error.' })
   public async updateCohortStatus(
-    @Param("cohortId") cohortId: string,
+    @Param('cohortId') cohortId: string,
```

ESLint

[error] Replace `'cohortId'` with `"cohortId"`

(prettier/prettier)